akirachix/Quantum-Informational
# added footer

Engineers Checklist
Summary of the Change

This PR updates the informational website footer to include new links and contact information.
Motivation and Context
To enhance user experience by providing easy access to important resources and contact options. Feedback from users indicated confusion about how to reach support and find additional information. This update aims to streamline navigation and improve overall usability.
Dependencies
What is this?
  This PR modifies the footer of the informational website to add links to support resources, privacy policy, and a contact form.

 Why this?
    To address user feedback regarding the difficulty of locating important resources.

List of updates:
 
Added links to support and contact resources.
Updated the privacy policy link.
Enhanced footer styling for better visibility.

Type of change

Improvement
Bug fix
New feature
Breaking change
This change requires a documentation update

How can it be tested?
Navigate to the website footer.
Check for the presence of new links (support, contact, privacy policy).
 Click each link to verify they direct to the correct pages.

Where is it documented?
https://nthenyamuenicynthia.atlassian.net/jira/software/projects/QP/boards/2
